filip/utils/cleanup.py

```diff
@@ -11,9 +11,10 @@
     ContextBrokerClient, \
     IoTAClient, \
     QuantumLeapClient
+from requests import Session
 
 
-def clear_context_broker(url: str, fiware_header: FiwareHeader):
+def clear_context_broker(url: str, fiware_header: FiwareHeader, session: Session = None ):
```

Comment:
Instead of adding Session as a parameter, I will suggest to use `cb_client: ContextBrokerClient = None`. And then reuse the given client below. This can be beneficial for maintenance, because if later we add or fix something in the client class, then we don't need to change here again

Reply:
And maybe make `url `no more as required variable